### PR TITLE
Show course detail for trainee

### PR DESCRIPTION
**Pull request by ngotungson**

### CODE REVIEW

File: app/views/user_courses/show.html.erb [hunk 1]
```diff
@@ -1,37 +1,40 @@
...
-        <div class="panel-body">
-          <%= @user_course.course.description %>
-        </div>
-      </div>
+<div class="col-md-4">
+  <div class="card">
+    <center>
+      <h3 class="course-name">
+        <%= @course.name %>
+        <span class="label label-<%= @course.status_label %>"><%= @course.status_content %></span>
```
Comment (cuongncFramgia) [inline app/views/user_courses/show.html.erb]: over  80

File: app/views/user_courses/_trainees_on_course.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      </thead>
+        <% trainees.each do |trainee| %>
+            <tr>
+              <td><%= link_to trainee.name, "#" %></td>
+              <td><%= trainee.email %></td>
+              <% if trainee.in_process? %>
+                <td><span class="label label-success"> <%= t ".in_process" %> </span></td>
+              <% else %>
+                <td><span class="label label-info"> <%= t ".free" %> </span></td>
+              <% end %>
```
Comment (phanbt58) [inline app/views/user_courses/_trainees_on_course.html.erb]: thieu the </tr>

**Comment by phanbt58** [comment]
ok

**Comment by huongnguyenmta** [comment]
OK

**Pull request merged**